### PR TITLE
RCPP frontend page

### PR DESCRIPTION
**Pull request by ywkim312**
The PR is for frontend for RCPP section

## Description
There should be somethings to be checked
1. there is no statue but just one map and table
2. what should be the columns for the tables
3. tree map? maybe not

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation and CHANGELOG accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main.tsx
Comment (pengyin-shan) [inline src/main.tsx:1]: we want both rcpp and acep page here

File: src/components/ProgramDrawer.tsx
Comment (pengyin-shan) [inline src/components/ProgramDrawer.tsx:1]: We want both ACEP and RCPP in program drawer

File: src/pages/RCPPPage.tsx [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                undefined,
+                {
+                    maximumFractionDigits: 2
+                }
+            )}M`;
+            setTotalBenefit(totalBenefitTmp);
+        }
+
+        // position="center"
+        // dy={-75}
```
Comment (pengyin-shan) [inline src/pages/RCPPPage.tsx:103]: feel free to remove the content that you won't need anymore

**Review by pengyin-shan — DISMISSED** [review]

**Comment by pengyin-shan** [comment]
Please note that this PR also includes some improvements I made to Title 1 and other pages.  Thus besides ACEP and RCPP,  these files will also include changes.

**Review by pengyin-shan — APPROVED** [review]

**Review by navarroc — APPROVED** [review]

**Pull request merged**